Makefile

```diff
@@ -35,6 +42,7 @@ install: install-extension install-headers
 install-downgrades:
 	$(MAKE) -C src/backend/distributed/ install-downgrades
```

Comment:
should we also have one for columnar ?

Reply:
The upgrades/downgrades are still installed like before, we didn't make changes to that yet.